src/main.js

```diff
@@ -54,3 +55,10 @@ function switchView(hiddenPage, visiblePage, hiddenButton1, hiddenButton2, visib
   visibleButton.classList.toggle('hidden')
 }
 switchView()
+
+function viewHome() {
```

Comment:
I am still curious if there is a better way for us to handle our functions. Is it better SRP to split them?